services/githubService.js

```diff
@@ -30,10 +30,19 @@ class GitHubService {
         return data;
     } 
     async getRepositoryForOrganization(org, repositoryName) {
-        const data = await this.octokit.rest.repos.listForOrg({
-            org: org
-        });
-        const repo = data.data.find(repo => repo.name === repositoryName);
+        let repos = [];
+        let page = 1;
+        while(true) {
```

Comment:
The variable page must be set to 1 and not 0 otherwise there will be duplicates inside the repos array. It seems that page 0 and 1 have the same elements. I have changed also the condition inside the while closure from != to == so the cycle can terminate when there are no more elements inside the API's result.